### PR TITLE
apps/measures/fields of action: making links on detail page lead to f…

### PR DESCRIPTION
**Pull request by khamui**
…iltered overview page

### CODE REVIEW

File: apps/measures/models.py [hunk 1]
```diff
@@ -518,9 +518,29 @@ class MeasuresDetailPage(Page):
...
 
     @property
-    def get_fields_of_action_display(self):
-        fields_of_action = [self.get_regenerative_management_display(),
-                            self.get_future_opportunities_for_all_display(),
-                            self.get_inclusive_shaping_of_urban_life_display(),
-                            self.get_facilitative_administration_display()]
-        return ', '.join(filter(None, fields_of_action))
+    def get_fields_of_action(self):
+        fields_of_action = self.get_selected_foa(
```
Comment (fuzzylogic2000) [inline apps/measures/models.py:522]: Could you add a docstring sth. like
`Return all selected fields_of_action in the form ....`
or `Return a dictianary of all selected measures and their sets`

File: apps/measures/models.py [hunk 2]
```diff
@@ -518,9 +518,29 @@ class MeasuresDetailPage(Page):
...
+        ) + self.get_selected_foa(
+            self.inclusive_shaping_of_urban_life, 'inc',
+            INCLUSIVE_SHAPING_OF_URBAN_LIFE
+        ) + self.get_selected_foa(
+            self.facilitative_administration, 'fac',
+            FACILITATIVE_ADMINISTRATION
+        )
+        return fields_of_action
+
+    def get_selected_foa(self, selected_foa_ids, kind_id, kind):
```
Comment (fuzzylogic2000) [inline apps/measures/models.py:537]: The `kind` did irritate me a bit. Can we rename that to maybe `set` or `category`? Or @philli-m has an idea how to name the groups or sets of fields of action?

File: apps/measures/models.py [hunk 3]
```diff
@@ -518,9 +518,29 @@ class MeasuresDetailPage(Page):
...
+        selected_foa = []
```
Comment (fuzzylogic2000) [inline apps/measures/models.py:538]: And a docstring here!

**Review by fuzzylogic2000 — COMMENTED** [review]
NIce! I just find the naming a bit confusing and some documentation would be good.

File: apps/measures/models.py [hunk 4]
```diff
@@ -518,9 +518,31 @@ class MeasuresDetailPage(Page):
...
 
     @property
-    def get_fields_of_action_display(self):
-        fields_of_action = [self.get_regenerative_management_display(),
-                            self.get_future_opportunities_for_all_display(),
-                            self.get_inclusive_shaping_of_urban_life_display(),
-                            self.get_facilitative_administration_display()]
-        return ', '.join(filter(None, fields_of_action))
+    def get_fields_of_action(self):
+        """Return a dictionary of selected measures and their categories."""
```
[the remark this answers is not in the document]
Comment (khamui) [reply apps/measures/models.py:522]: @fuzzylogic2000 ahh..true, sorry. Yes it should be `selected fields of action` instead

**Review by fuzzylogic2000 — APPROVED** [review]
Did you want to change it to subset or should we merge like this? I am fine eith either.

**Comment by fuzzylogic2000** [comment]
> > Did you want to change it to subset or should we merge like this? I am fine eith either.
> 
> oh..i forgot, will change it!

You didn't push it, right? :stuck_out_tongue_closed_eyes:

**Comment by khamui** [comment]
@fuzzylogic2000 oh true....uppsala

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**